packages/core/src/data-grid/data-grid-lib.ts

```diff
@@ -294,7 +302,7 @@ export function prepTextCell(
     return result;
 }
 
-export function drawTextCell(args: BaseDrawArgs, data: string) {
+export function drawTextCell(args: BaseDrawArgs, data: string, cell: BaseGridCell) {
```

Comment:
I'd rather this pass just the data and contentAlign. Passing the entire cell passes more data than we potentially want for these functions to have. Minor issue but this is just a general preference. Then again... some functions in this project have just stupid numbers of args... so what do I know.